Makefile

```diff
@@ -258,7 +255,7 @@ update:
 # set up prj.conf file
 -.PHONY: setup
 setup:
-	@echo "# This is a generated file" > prj.conf
+	@echo "# This is a generated file" >> prj.conf
```

Comment:
This should be the top line of the file, so if it's not now, can you rearrange things so it will be?